index.html

```diff
@@ -1,339 +1,330 @@
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="c-primary-text">
+                    <p>A daily selection of privately personalized reads: no 
+                        account or sign-ups required.
+                    </p>
+                </div>
+                <div class="tags">
+                    <div><p>html</p></div>
+                    <div><p>css</p></div>
+                    <div><p>javascript</p></div>
+                </div>
+                <div class="see-proj">
+                    <button id="wkbtn3" type="button">See Project</button>
+                </div>
+            </div>
         </div>
-      </section>
-
-      <section class="works-temp1 works4">
-        <img src="images/SnapshootPortfolio(4).png" class="pic-temp1" alt="" />
-        <div class="works-info4">
-          <h1 class="tonic">
-            Multi-Post <br />
-            Stories
-          </h1>
-          <ul class="work-info">
-            <li class="canopy">CANOPY</li>
-            <li class="back-end-dev">
-              <img src="images/Counter(1).png" alt="" />
-              <span>Back End Dev</span>
-            </li>
-            <li class="back-end-dev">
-              <img src="images/Counter(1).png" alt="" />
-              <span>2015</span>
-            </li>
-          </ul>
-          <p class="proj-info">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="prog-lang">
-            <li class="lang-styles">HTML</li>
-            <li class="lang-styles">Css</li>
-            <li class="lang-styles">JavaScript</li>
-          </ul>
-          <button type="button" class="btn">See Projects</button>
+        <div class="card">
+            <div class="project1">
+                <img src="./images/Snapshoot-4Portfolio.svg" alt="project1 snap">
+            </div>
+            <div class="card-content">
+                <div>
+                    <h3>Tonic</h3>
+                </div>
+                <div class="client">
+                    <p>CANOPY</p>
+                    <ul>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="c-primary-text">
+                    <p>A daily selection of privately personalized reads: no 
+                        account or sign-ups required.
+                    </p>
+                </div>
+                <div class="tags">
+                    <div><p>html</p></div>
+                    <div><p>css</p></div>
+                    <div><p>javascript</p></div>
+                </div>
+                <div class="see-proj">
+                    <button id="wkbtn4" type="button">See Project</button>
+                </div>
+            </div>
         </div>
-      </section>
     </section>
```

Comment:
- [ OPTIONAL ] Great work on implementing popup window 👌💯 but according to the requirements, The works/projects section should also be created dynamically using the information stored in that JavaScript object. So, kindly remove all of the HTML in that section and create them dynamically using JS when the page loads. 👌
> I will ask for an optional change for this because the previous reviewers didn't mention this issue. but I strongly recommend fixing it because it's one of the requirements. 💯

![1](https://user-images.githubusercontent.com/45231276/181671394-98c83a6c-7379-4e9d-b016-22ff67e91b36.PNG)